pkg/leeway/scripts_test.go

```diff
@@ -50,155 +70,154 @@ func TestScriptArgs(t *testing.T) {
 }
 
 func TestWorkingDirLayout(t *testing.T) {
-	runDUT()
+	testutil.RunDUT()
+
+	setup := &testutil.Setup{
+		Components: []testutil.Component{
+			{
+				Location: "scripts",
+				Packages: []leeway.Package{genericPackage},
+				Scripts: []leeway.Script{
+					{
+						Name:          "pwd-origin",
+						WorkdirLayout: "origin",
+						Dependencies:  []string{fmt.Sprintf(":%s", genericPackage.Name)},
+						Script:        `pwd && find .`,
+					},
+					{
+						Name:          "pwd-packages",
+						WorkdirLayout: "packages",
+						Dependencies:  []string{fmt.Sprintf(":%s", genericPackage.Name)},
+						Script:        `pwd && find .`,
+					},
+				},
+			},
+		},
+	}
 
-	tests := []*CommandFixtureTest{
+	tests := []*testutil.CommandFixtureTest{
+		// Shows that even though it depends on a package, it isn't copied into the working directory
 		{
 			Name:              "origin",
 			T:                 t,
 			Args:              []string{"run", "scripts:pwd-origin"},
 			ExitCode:          0,
 			NoNestedWorkspace: true,
-			FixturePath:       "fixtures/scripts.yaml",
-			StdoutSub: `.
-./BUILD.yaml`,
+			Fixture:           setup,
+			StdoutSubs: []string{`.
+./BUILD.yaml`},
 		},
+		// Shows that the dependency is copied into the working directly, and that nothing else is present
 		{
 			Name:              "packages",
 			T:                 t,
 			Args:              []string{"run", "scripts:pwd-packages"},
 			ExitCode:          0,
 			NoNestedWorkspace: true,
-			FixturePath:       "fixtures/scripts.yaml",
-			StdoutSub: `.
-./pkgs-generic--something`,
+			Fixture:           setup,
+			StdoutSubs: []string{`.
+./scripts--something`},
 		},
-		// 		{
```

Comment:
I assumed these were related to nested workspaces, so decided to not try to port them over